Code/Framework/AzCore/AzCore/Debug/MemoryProfiler.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#ifndef AZ_PROFILE_MEMORY_ALLOC
+// No other profiler has defined the performance markers AZ_PROFILE_MEMORY_ALLOC (and friends), fall back to a Driller implementation (currently empty)
```

Comment:
Is the comment about Driller still valid here? I thought Drilller is redcoded?

Reply:
It is but that code still exists unfortunately. A future PR will remove driller completely

Reply:
This also means that we loose our ability to track memory at a lower level. External profilers will only see the larger allocations the engine makes to the OS. Driller should definitely be removed but a quick follow up with a replacement is needed, even if it's nothing more than a Python script that does some basic drawing.